### DISCUSSION TITLE
Search in OneNote

### DISCUSSION THREAD

**Discussion by igrokit** [adirh3/Fluent-Search#765] [Ideas]
"PowerToys Run" supports searching in OneNote files.
i.e.

o: powertoys 

to search your local OneNote notebooks for pages containing "powertoys" (see https://learn.microsoft.com/en-us/windows/powertoys/run).
 Would be great if Fluent Search supports this as well.
Or does it already?

Dan

![17-01-2023_10-18-45](https://user-images.githubusercontent.com/15889404/212858767-d76cfa65-1591-47a1-a551-a173d40f09db.png)

**Comment by Shlomo-machsivaya** [comment]
Or just search for sections and pages in OneNote
